cfgov/ask_cfpb/tests/test_search.py

```diff
@@ -82,7 +85,7 @@ def test_AnswerPage_search(self):
             live=True,
         )
         self.ROOT_PAGE.add_child(instance=test_answer_page)
-        self.rebuild_elasticsearch_index("ask_cfpb", stdout=StringIO())
+        self.rebuild_elasticsearch_index("test-ask-cfpb", stdout=StringIO())
```

Comment:
Oops, I didn't see these the first time. Can we use `AnswerPageDocument.Index.name` here too?

```suggestion
        self.rebuild_elasticsearch_index(
            AnswerPageDocument.Index.name, 
            stdout=StringIO()
        )
```